lib/services/runtime/handlers/interaction.ts

```diff
@@ -78,8 +79,10 @@ export const InteractionHandler: HandlerFactory<Node.Interaction.Node, typeof ut
       variables.merge(utils.mapSlots(variableMap, slots));
     }
 
+    if (nextId !== undefined) return nextId;
```

Comment:
very important that you do this:
```suggestion
    if (nextId !== undefined) {
      runtime.turn.delete(T.REQUEST);
      return nextId;
    }
```
if you don't delete the request, the next interaction block will still try to handle it.